python/doc/examples/reliability_sensitivity/reliability/plot_subsetSampling.py

```diff
@@ -220,7 +220,7 @@
 graph.setXTitle(r'$x_1$')
 graph.setYTitle(r'$x_2$')
 
-Show(graph)
+ot.Show(graph)
```

Comment:
could we use viewer instead ? that's what we promote in all other examples except this one (maybe we should consider deprecating it someday)

Reply:
Done. What does Show do differently than the viewer anyway?

Reply:
it just calls viewer :)